core/Piranha.Manager/assets/src/js/piranha.content.js

```diff
@@ -0,0 +1,482 @@
+    el: "#content",
+    data: {
+        contentType: null,
+        onSave: null,
+        prevId: null,
+
+        // Features
+        features: {
+            useAltTitle: false,
+            useBlocks: false,
+            useCategory: false,
+            useComments: false,
+            usePrimaryImage: false,
+            useExcerpt: false,
+            useHtmlExcerpt: false,
+            usePublish: false,
+            useTags: false,
+            useTranslations: false
+        },
+
+        // Model data
+        id: null,
+        languageId: null,
+        languageTitle: null,
+        parentId: null,
+        typeId: null,
+        typeTitle: null,
+        groupId: null,
+        groupTitle: null,
+        position: null,
+        meta: null,
+        comments: null,
+        permissions: {
+            selectedPermissions: [],
+            permissions: []
+        },
+        routes: null,
+        taxonomies: null,
+        title: null,
+        altTitle: null,
+        slug: null,
+        primaryImage: null,
+        excerpt: null,
+        state: null,
+        isReadOnly: false,
+        isScheduled: false,
+        published: null,
+        publishedTime: null,
+
+        blocks: [],
+        editors: [],
+        regions: [],
+        languages: [],
+        sections: [],
+
+        // State members
+        loading: true,
+        saving: false,
+        savingDraft: false,
+        selectedRegion: "",
+        selectedSetting: "uid-settings"
+    },
+    computed: {
+        contentRegions: function () {
+            return this.regions.filter(function (item) {
+                return item.meta.display != "setting" && item.meta.display != "hidden";
+            });
+        },
+        settingRegions: function () {
+            return this.regions.filter(function (item) {
+                return item.meta.display === "setting";
+            });
+        },
+        primaryImageUrl: function () {
+            if (this.primaryImage.media != null) {
+                return piranha.utils.formatUrl("~/manager/api/media/url/" + this.primaryImage.id + "/448/200");
+            } else {
+                return piranha.utils.formatUrl("~/manager/assets/img/empty-image.png");
+            }
+        },
+        isExcerptEmpty: function () {
+            return piranha.utils.isEmptyText(this.excerpt);
+        },
+        isPreviewActive: function () {
+            return !this.features.useBlocks &&
+                this.regions.length === 0 &&
+                this.editors.length === 0;
+        },
+        isSettingsActive: function () {
+            return !this.isPreviewActive ||
+                this.meta !== null ||
+                this.features.usePublish ||
+                this.features.useAltTitle;
+        },
+        metaPriorityDescription: function() {
+            var description = piranha.resources.texts.important;
+            if (this.metaPriority <= 0.3)
+                description = piranha.resources.texts.low;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/PiranhaCMS/piranha.core/pullRequest?prid=7918869)
```suggestion
                {description = piranha.resources.texts.low;}
```